python/fastquant/indicators/custom.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Import standard library
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+from pkg_resources import resource_filename
```

Comment:
seems `resource_filename` is not used anymore

Reply:
Fixed!